### PR TITLE
Blog post: Enhance OpenAI with Binary Quantization

### PR DESCRIPTION
**Pull request by mjang**

### CODE REVIEW

File: qdrant-landing/content/blog/binary-quantization-openai.md
Comment (mjang) [inline qdrant-landing/content/blog/binary-quantization-openai.md:37]:
```suggestion
### Multilingual Support
```

File: qdrant-landing/content/blog/binary-quantization-openai.md
Comment (mjang) [inline qdrant-landing/content/blog/binary-quantization-openai.md:74]:
```suggestion
```

File: qdrant-landing/content/blog/binary-quantization-openai.md
Comment (mjang) [inline qdrant-landing/content/blog/binary-quantization-openai.md:81]:
```suggestion
To analyze the impact of rescoring (`True` or `False`), we compared results across different model configurations and search limits. Rescoring sets up a more precise search, based on results from an initial query.
```

**Review by sabrinaaquino — APPROVED** [review]
I've added a few modifications. Lgtm but would like a final approval.

File: qdrant-landing/content/blog/binary-quantization-openai.md
Comment (mjang) [inline qdrant-landing/content/blog/binary-quantization-openai.md:35]: I'm not sure if we should refer to the Jupyter notebook or any other instructions _until_ @NirantK merges the `openai-3` branch into the `examples` repository.

(Maybe we should hold off merging until after Nirant merges his work into the `examples` repo.)

If we do, we'll have to change the link after the merge. And I think we should start with a different page:

```suggestion
You can also try out these techniques as described in [Binary Quantization OpenAI](https://github.com/qdrant/examples/blob/openai-3/binary-quantization-openai/README.md), which includes Jupyter notebooks.
```

After Nirant merges the change, the tech would change to:

```suggestion
You can also try out these techniques as described in [Binary Quantization OpenAI](https://github.com/qdrant/examples/blob/master/binary-quantization-openai/README.md), which includes Jupyter notebooks.
```

File: qdrant-landing/content/blog/binary-quantization-openai.md
Comment (mjang) [inline qdrant-landing/content/blog/binary-quantization-openai.md:41]:
```suggestion
These models include multilingual support in over 100 languages. The transition from text-embedding-ada-002 to text-embedding-3-large has led to a significant jump in performance scores (from 31.4% to 54.9% on MIRACL).
```

**Review by mjang — COMMENTED** [review]
Open question: should we include a link to the Jupyter notebook examples at all before Nirant merges.

**Review by sabrinaaquino — APPROVED** [review]

**Comment by sabrinaaquino** [comment]
Let's not include the notebook as it's not complete yet. We can revisit this later.

**Pull request merged**
